### DISCUSSION TITLE
[QUESTION] Multiple Camera's on single device

### DISCUSSION THREAD

**Discussion by nathan-swidget** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1705] [Q&A]
Hey,

I am working on a product in which one device has multiple camera's/feeds. Is it possible for the client to specify which feed they intend to view? What would be the appropriate approach to handle this within WebRTC, or would the intended feed need to be specified outside of the WebRTC protocol?

Thanks

**Comment by niyatim23** [comment]
I think that would be outside of the WebRTC protocol. Are you planning on using Gstreamer with RTSP?
